### PR TITLE
Add CODE_OF_CONDUCT.md

### LINKED ISSUE #8: Code of conduct
The project is currently missing the Code of Conduct file which is important for open source project
So can I go for it @97shivank

### PR DESCRIPTION
**Pull request by Halix267**
fixes #8 
Adding  CODE_OF_CONDUCT file

### CODE REVIEW

**Comment by Halix267** [comment]
@97shivank Plzz review it

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing [mailto:Daicience](sudeepsidhu1012@gmail.com).
```
Comment (97shivank) [inline CODE_OF_CONDUCT.md]: kindly edit this line.

Comment (Halix267) [reply CODE_OF_CONDUCT.md]: ok wait

Comment (Halix267) [reply CODE_OF_CONDUCT.md]: kindly provide your email address

Comment (97shivank) [reply CODE_OF_CONDUCT.md]: 97shivank@gmail.com

Comment (Halix267) [reply CODE_OF_CONDUCT.md]: done

**Pull request merged**